charts/pega/values.yaml

```diff
@@ -110,11 +110,29 @@ global:
         # Inactivity time after which requestor is passivated
         passivationTimeSec: 900
 
+
       service:
         # For help configuring the service block, see the Helm chart documentation
         # https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#service
         port: 80
         targetPort: 8080
+        tlsport: 443
+        tlstargetPort: 8443
+
+      # to enable TLS between ingress/loadbalancer and the backend, set the following:
+      tls:
+        enabled: false
+        keystore: ""
+        keystorepassword: ""
+        # set the value of CA certificate here in case of baremetal/openshift deployments - CA certificate should be in base64 format
+        cacertificate: ""
+        # set enabled=true, only if addon chart is deployed and traefik is enabled, else set to false
+        traefik:
+          enabled: false
```

Comment:
We recommend and default `traefik` for k8s based deployments. Instead of having another enabled flag for this section, Can this be derived from the `provider` variable like [here](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/templates/_pega-service.tpl#L14-L25)